internal/resource/sysfs-device.go

```diff
@@ -0,0 +1,66 @@
+/**
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+**/
+
+package resource
+
+import (
+	"fmt"
+
+	"github.com/NVIDIA/go-nvlib/pkg/nvpci"
+)
+
+type vfioDevice struct {
+	*nvpci.NvidiaPCIDevice
```

Comment:
let's rather make this a private member.

Reply:
Fixed